README.md

```diff
@@ -8,43 +8,151 @@ Please create a branch called `homework`. Then, create a file named `homework.js
   <li>Create a function that will get the sum of the numbers between 1 and n and return the answer
   <br>
     <code>summation(5) // should return 15 because 1+2+3+4+5=15</code>
+     
+        
+const sumNum= function (){
+
+const arr = [1,2,3,4,5];
+
+let n = 5;
+
+var res = (n * (n+1)) / 2;
+
+console.log( arr.join ("+") +" the sum it's " +" = " + res);
+
+}
+sumNum();
+
+
   </li>
   <li>Create a program to get the sum of all the even numbers in a group
   <br>
     <code > summationEven(5) // should return 6 because 2+4=6</code>
+
+
+function findSum() {
+
+ 
+  
+  let sum = 0;
+  for(let i = 0; i <= 5; i++){
+    if(i %2 ===0 ) {
+      sum += i;
+    }
+  }
+  return sum
+}
+
+console.log(`sum is ${findSum()}`)
+
   </li>
     
   <li>Create a function to get the average of a group of numbers 
   <br>
     <code> avg([8, 2, 2, 4]) // should return 4</code>
+    let values = [8, 2, 2, 4];
+let sum = values.reduce((previous, current) => current += previous);
+let avg = sum / values.length;
+
+console.log(values.join(",")+` the avg for this numbers = ${avg}`);
+
   </li>
   <li>Create a function to reverse the letters in a word
   <br>
     <code > reverse("caterpillar") // should return "rallipretac"</code>
+
+
+    function reverseNumber(n)
+{
+	n = n + "";
+	return n.split("").reverse().join("");
+}
+console.log(reverseNumber("caterpillar"));
   </li>
   <li>Create a function that takes an array of words and combines them with a dash
   <br>
     <code > addDashes(['test1', 'test2', 'test3']) // should return "test1-test2-test3"</code>
+
+    function combines(){
+
+  const addDashes= ["test1","test2","test"];
+
+
+  console.log(addDashes.join("-"));
+}
+combines();
   </li>
   <li>Function that will count up to a number and back down and return a string of the climb
   <br>
     <code> countUpAndDown(3) // should return "1 2 3 2 1"</code>
+    function countdown(i) {
+answer = '';
+while( i > 1) {
+answer = answer + i.toString() + ' ';
+i--;
+}
+
+while( i <= 3) {
```

Comment:
You hardcoded `3` here as your loop condition, but what happens if I call the function with another number not `3`? Consider `countdown(4)` or `countdown(6)`?